### PR TITLE
HW and exercise done

### PR DESCRIPTION
**Pull request by hakanErgin**
the excersize is done

### CODE REVIEW

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,143 @@
...
+  //iterating thru books, creating h2/ul/li's
+  for (let id in bookTitles) {
+    let header = document.createElement('h2');
+    let bookNumber = document.createTextNode(id);
+    let newList = document.createElement('ul');
+    let image = document.createElement("img");
+    image.setAttribute("src", imgList[id]);
+    
+    // li
+    for (let data in bookTitles[id]) {
```
Comment (segersrobbert) [inline Week1/homework/app.js]: nested for loop, great!

File: Week1/homework/app.js [hunk 2]
```diff
@@ -1,11 +1,143 @@
...
+    }
+    header.appendChild(bookNumber);
+    document.body.appendChild(header);
+    document.body.appendChild(newList);
+    newList.appendChild(image);
+  }
+}
+makeList();
+
+// array for the first step
```
Comment (bitsagarob) [inline Week1/homework/app.js:95]: you can remove old commented code

File: Week1/homework/style.css [hunk 3]
```diff
@@ -1 +1,13 @@
-/* add your styling here */
\ No newline at end of file
+body{
+    margin:2ch 15% 2ch 15%;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
+}
+
```
Comment (bitsagarob) [inline Week1/homework/style.css:5]: could use some more styling

File: Week1/homework/app.js [hunk 4]
```diff
@@ -1,11 +1,143 @@
...
+  "book9": "./img/warandpeace.jpg",
+  "book10": "./img/things_fall_apart.jpg"
+}
+
+// iterating and making lists with the objects
+function makeList() {
+
+  //iterating thru books, creating h2/ul/li's
+  for (let id in bookTitles) {
+    let header = document.createElement('h2');
```
Comment (segersrobbert) [inline Week1/homework/app.js]: use const for variables that don't change

**Review by bitsagarob — APPROVED** [review]
Great job! few remarks

**Pull request closed without merging**